cmd/livepeer_cli/wizard_transcoder.go

```diff
@@ -61,7 +66,8 @@ func (w *wizard) promptOrchestratorConfig() (float64, float64, *big.Int, string)
 		return in, nil
 	})
 
-	return blockRewardCut, feeShare, pricePerSegment, serviceURI
+	// TODO (nico): Remove pricePerSegment after contract updates
```

Comment:
Can remove this comment?

Reply:
fixed in https://github.com/livepeer/go-livepeer/pull/993/commits/54863dce0b699a81e16eb27e09bf64b6b28e79a8